Makefile

```diff
@@ -87,8 +87,11 @@ gencatalog:
 	rm site/content/en/guides/consumer/function/catalog/*/_index.md
 	(cd site/content/en/guides/consumer/function/catalog/catalog && npm run gen-docs)
 
-servedocs:
-	(cd site && go run github.com/gohugoio/hugo server)
+run-site:
```

Comment:
it's better to change the order here: <noun>-<verb>.

This way it's easier to find all the verbs for a given noun.

`site-run-server`
`site-check`